Remotion/Data/DomainObjects.Validation.UnitTests/DomainObjectAttributesBasedValidationPropertyRuleReflectorTests/DomainObject_DOABCPRRTest.cs

```diff
@@ -363,9 +362,7 @@ public void GetMetaValidationRules_NullableStringPropertyAttribute ()
     {
       var result = _propertyWithNullableStringPropertyAttributeReflector.GetMetaValidationRules().ToArray();
 
-      Assert.That(result.Count(), Is.EqualTo(1));
-      Assert.That(result[0], Is.TypeOf(typeof(RemotionMaxLengthPropertyMetaValidationRule)));
-      Assert.That(((RemotionMaxLengthPropertyMetaValidationRule)result[0]).MaxLength, Is.EqualTo(10));
+      Assert.That(result.Any, Is.False);
```

Comment:
```suggestion
      Assert.That(result, Is.Empty);
```

Reply:
All tests in this file use the any -> is.false syntax. I'm not sure that changes anything but I'd like you to know.

Reply:
Ah, I see, there's a couple more in this and the other file. Hmm... no, we can still udpate the current tests. I would also be okay with changing the remaining couple of assertions to Is.Empty in these two files.